packages/core/src/state-management/repository.ts

```diff
@@ -487,28 +496,40 @@ export class Repository {
     // We also skip CACAO expiration checking during this initial load as its possible
     // that the CommitID we are being asked to load may in fact be an anchor commit with
     // the timestamp information that will reveal to us that the CACAO didn't actually expire.
-    const base$ = await this.load(commitId.baseID, opts, false)
```

Comment:
this was the issue.  The "false" in the third arg here was to `skipCacaoExpiration` checks, and if the node already has the state stored locally, that worked.  But if the node didn't already have the stream in its state store, and the genesis commit has an expired CACAO, this will fail with a CACAO error when trying to build the initial state from the genesis commit.

The "right" fix would be to thread the `skipCacaoExpiration` down all the way through all the layers of commit application, so that at the point where the genesis commit is applied we can inform it to not worry about cacao expiration.  But that would require a much bigger code change.  So instead this is kind of a hack to just try our best to get the information from existing state in the state store if we have it, but if we don't have it then just go ahead and load the state directly at the CommitID given.